apps/guide/views.py

```diff
@@ -0,0 +1,14 @@
+from django.views.generic import TemplateView
+from wagtail.core.models import Site
+
+
+class RobotsView(TemplateView):
```

Comment:
I’m surprised we’d want a view inside the `guide` app, but this feels simple enough to refactor later on so let’s leave this as-is for now.